### PR TITLE
Add Unit tests of max failures and percentage failure rules

### PR DESCRIPTION
**Pull request by drgarcia1986**
shameless copy from `luizalabs/django-toolkit`

### CODE REVIEW

File: lasier/circuit_breaker/rules/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from .base import RuleBase  # noqa
```
Comment (barioto) [inline lasier/circuit_breaker/rules/__init__.py]: Think we can change this class to `BaseRule` ?
IMO its better to read and understand its meaning

File: tests/circuit_breaker/rules/test_max_failures_rule.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        )
+
+    def test_max_failures_rule_should_return_rule_instance(self, rule):
+        assert isinstance(rule, RuleBase)
+
+    def test_should_not_open_circuit(self, rule):
+        assert rule.should_open_circuit(
+            total_failures=5,
+            total_requests=5
+        ) is False
```
Comment (luizrabachini-luizalabs) [inline tests/circuit_breaker/rules/test_max_failures_rule.py]: Maybe we can test the limits using `total_failures=1` and `total_failures=10` with `parametrize`, what you think?

File: tests/circuit_breaker/rules/test_max_failures_rule.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+    def test_should_open_circuit(self, rule):
+        assert rule.should_open_circuit(
+            total_failures=20,
+            total_requests=5
+        ) is True
```
Comment (luizrabachini-luizalabs) [inline tests/circuit_breaker/rules/test_max_failures_rule.py]: A similar suggestion here to use parametrize with `total_failures=11` and `total_failures=20`.

Comment (drgarcia1986) [reply tests/circuit_breaker/rules/test_max_failures_rule.py]: Hi @luizrabachini-luizalabs, thanks, I changed to only one test to check the state of the circuit by total failures, tell me what you think about it

**Comment by drgarcia1986** [comment]
@barioto @luizrabachini-luizalabs  UPDATED

File: tests/circuit_breaker/rules/test_max_failures_rule.py (hunk 3 above)
Comment (luizrabachini-luizalabs) [reply tests/circuit_breaker/rules/test_max_failures_rule.py]: Looks good! Thank you!

**Pull request merged**